src/info/info_client.rs

```diff
@@ -41,6 +41,8 @@ pub enum InfoRequest {
         user: H160,
     },
     Meta,
+    #[serde(rename = "spotMeta")]
```

Comment:
You shouldn't need to rename as this is the default name

Reply:
Realized that it was redundant. Thanks. Made the change.